cluster/kubemark/config-default.sh

```diff
@@ -22,7 +22,7 @@
 GCLOUD=gcloud
 ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-4}
-NUM_MINIONS=${NUM_MINIONS:-100}
+NUM_MINIONS=${NUM_MINIONS:-4000}
```

Comment:
won't this break `kubernetes-kubemark-gce`? or at least cause it to try to use 4000 minions too, which may break it?